tools/transformer/src/discriminator.ignore.json

```
@@ -1,3 +1,3 @@
 {
-  "ignoreModels": ["ApiLimitView"]
+  "ignoreModels": [""]
```

wouldn't it be better to do
```suggestion
  "ignoreModels": []
```